src/__tests__/storeStress-test.js

```diff
@@ -18,6 +18,7 @@ describe('StoreStress', () => {
   beforeEach(() => {
     bridge = global.bridge;
     store = global.store;
+    store.collapseNodesByDefault = false;
```

Comment:
This default preserves the behavior of the `Store` at the time when the snapshot tests were originally written. We may want to add additional stress tests for the collapse-by-default behavior, but I wanted to reduce churn on this PR.